Sources/BraintreePayPal/BTPayPalVaultBaseRequest.swift

```diff
@@ -66,6 +72,10 @@ import BraintreeCore
 
             vaultParameters["shipping_address"] = shippingAddressParameters
         }
+        
+        if let fallbackURLScheme {
+            vaultParameters["merchant_app_fallback_url_scheme"] = fallbackURLScheme
```

Comment:
I think we discussed internally setting this on the `BTPayPalRequest`. That way when deeplinks are enabled there a second merchant upgrade isn't needed. If that's the case we may only need it on the base request class since it's set in the init, but not 100% sure.